clients/client-accessanalyzer/tsconfig.cjs.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig",
   "compilerOptions": {
-    "outDir": "dist-cjs"
+    "outDir": "dist-cjs",
+    "noCheck": true
```

Comment:
This needs to be done at codegen level

* https://github.com/smithy-lang/smithy-typescript/blob/main/smithy-typescript-codegen/src/main/resources/software/amazon/smithy/typescript/codegen/tsconfig.cjs.json
* https://github.com/smithy-lang/smithy-typescript/blob/main/smithy-typescript-codegen/src/main/resources/software/amazon/smithy/typescript/codegen/tsconfig.es.json